app/assets/stylesheets/default_theme/date_selector.scss

```diff
@@ -0,0 +1,84 @@
+// scss-lint:disable all
```

Comment:
I know we already about about this in person, but can you maybe add a comment here explaining why we have to do this (for future reference)?